File: CafeEase/CafeEase.Services/Database/Payment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CafeEase.Services.Database
{
    public class Payment
    {
        public int Id { get; set; }
        public string Method { get; set; } = null!;
        public string Status { get; set; } = null!;
        public int OrderId { get; set; }
        public virtual Order Order { get; set; } = null!;
    }
}

File: CafeEase/UI/cafeease_desktop/lib/models/payment.dart
class Payment {
  final int? id;
  final String? method;
  final String? status;
  final int? orderId;

  Payment({this.id, this.method, this.status, this.orderId});

  factory Payment.fromJson(Map<String, dynamic> json) {
    return Payment(
      id: json['id'],
      method: json['method'],
      status: json['status'],
      orderId: json['orderId'],
    );
  }
}
